Extensions/BBText/JsExtension.js

```diff
@@ -0,0 +1,588 @@
+              '',
+              `${property.iconPath}.png`,
+              `${property.iconPath}.png`
+            )
+            .addParameter(
+              'object',
+              _(`${objectName} object`),
+              objectName,
+              false
+            )
+            .getCodeExtraInformation()
+            .setFunctionName(`get${property.functionName}`);
+        }
+
+        // Add all the generic SETTERS
+        if (parameterType === 'number' || parameterType === 'string') {
+          const expressionType =
+            parameterType === 'number' ? 'expression' : 'string';
+          gdObject
+            .addAction(
+              `Set${property.functionName}`,
+              _('Set ') + property.description,
+              _('Set ') + property.description,
+              _('Do _PARAM1__PARAM2_ to the ') +
+                property.description +
+                _(' of _PARAM0_'),
+              '',
+              `${property.iconPath}.png`,
+              `${property.iconPath}.png`
+            )
+            .addParameter(
+              'object',
+              _(`${objectName} object`),
+              objectName,
+              false
+            )
+            .addParameter('operator', _("Modification's sign"), '', false)
+            .addParameter(expressionType, _(property.paramLabel), '', false)
+            .getCodeExtraInformation()
+            .setFunctionName(`set${property.functionName}`)
+            .setManipulatedType(parameterType)
+            .setGetter(`get${property.functionName}`);
+        } else {
+          // Setter doesnt have a +- modifier (Color, alignment, etc)
+          gdObject
+            .addAction(
+              `Set${property.functionName}`,
+              _('Set ') + property.description,
+              _('Set ') + property.description,
+              _('Set ') + property.description + _(' of _PARAM0_ to _PARAM1_'),
+              '',
+              `${property.iconPath}.png`,
+              `${property.iconPath}.png`
+            )
+            .addParameter(
+              'object',
+              _(`${objectName} object`),
+              objectName,
+              false
+            )
+            .addParameter(
+              parameterType,
+              _(property.paramLabel),
+              property.options ? property.options : '',
+              false
+            )
+            .getCodeExtraInformation()
+            .setFunctionName(`set${property.functionName}`)
+            .setGetter(`get${property.functionName}`);
+        }
+
+        // Add compare Conditions via getters
+        if (parameterType === 'string' || parameterType === 'number') {
+          const propExpressionType =
+            parameterType === 'string' ? 'string' : 'expression';
+          gdObject
+            .addCondition(
+              `Is${property.functionName}`,
+              property.description,
+              _('Check if the ') + property.description + _(' equals a value'),
```

Comment:
`_("Compare the value of ") + property.description` seems a bit better - though again I'm worried of these concatenation that will result in bad quality translation :'(

Reply:
The upside to this is that once we have it right once - it will be right everywhere the utility function is used - so there will be less work for translators